windup.info.yml

```diff
@@ -0,0 +1,20 @@
+name: Windup
+description: 'A clean theme for Drupal 8'
+core: 8.x
+type: theme
+screenshot: screenshot.png
+stylesheets-remove:
+  - core/assets/vendor/normalize-css/normalize.css
+  - core/modules/system/css/system.theme.css
+  - core/modules/system/css/system.menus.css
+
+libraries:
+  - windup/global-styling
+#  - windup/global-js
```

Comment:
I'm fine with leaving this here for now, especially since people may still be wtfing at the way things need to be done for a while. This is a good hint.